libglusterfs/src/inode.c

```diff
@@ -137,6 +137,21 @@ inode_table_prune(inode_table_t *table);
 void
 fd_dump(struct list_head *head, char *prefix);
 
+/* Calculate index based on the inode_table root_level
+   and xlator_id
+*/
+static int
+inode_get_ctx_index(inode_t *inode, xlator_t *xlator)
```

Comment:
We don't need the inode for anything. It would be better to pass the inode table.

Reply:
done